### ISSUE TITLE
Is there any way to send an email from the temporary email?

### ISSUE THREAD

**Issue by Zeputee** [ncjones/python-guerrillamail#19]
The guerrillamail website offers this and says that it's a beta feature; is there any way of sending emails from temporary emails with python-guerrillamail?

**Comment by ncjones** [comment]
No. This tool is geared towards receiving system generated emails. If you need to send messages on behalf of a user then you probably should use a different tool. Probably a test account with a real email provider would be a better choice. See https://docs.python.org/3/library/email.examples.html.

Having said that, if newer versions of Guerillamail support sending email then there is no reason why it should not also be supported by this client library. I don't have capacity to add features but I will accept good quality pull requests.
